build-tools/packages/build-cli/src/lib/package.ts

```diff
@@ -33,53 +46,86 @@ import { isReleaseGroup, ReleaseGroup, ReleasePackage } from "../releaseGroups";
 // eslint-disable-next-line max-params
 export async function npmCheckUpdates(
     context: Context,
-    releaseGroup: ReleaseGroup | ReleasePackage,
+    releaseGroup: ReleaseGroup | ReleasePackage | undefined,
     depsToUpdate: ReleasePackage[] | RegExp[],
-    bumpType: "patch" | "minor" | "current",
+    releaseGroupFilter: ReleaseGroup | undefined,
```

Comment:
Would it make sense for this to be an array of groups to filter? Or is there not a use-case for filtering out multiple?

Reply:
Hmmm, I can't think of a scenario for that at the moment, so I'll leave this for now. If a scenario comes up we can change it.